src/resources/MLModels/MLModels.ts

```diff
@@ -0,0 +1,41 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {CreateMLModelOptions, MLModel, MLModelInfo} from './MLModelsInterfaces';
+
+export default class MLModels extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/models`;

```

Comment:
```suggestion
    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning`;
```
I think the base url could be a little more generic so that you don't have to define `createUrl` and `infoUrl`